test/test_mps.py

```diff
@@ -10,10 +10,12 @@
 import torch.nn as nn
 import torch.nn.functional as F
 import itertools
+from torch._six import inf
 from torch.nn import Parameter
 from torch.testing._internal.common_utils import run_tests, TestCase, download_file, TEST_WITH_UBSAN
 import torch.backends.mps
 from torch.distributions import (Uniform)
+from scipy import stats
```

Comment:
I don't think scipy is a dependency in the CI.
You can use `TEST_SCIPY` from `common_utils` to know if you should run tests requiring scipy.